### PR TITLE
feat: account recover with email

### PR DESCRIPTION
**Pull request by hugomrdias**
Theres also a commit with changes to rename access to access-client

### CODE REVIEW

File: packages/access-api/src/service/index.js [hunk 1]
```diff
@@ -18,13 +22,50 @@ export function service(ctx) {
     },
 
     account: {
-      info: Server.provide(Account.info, async ({ capability }) => {
+      info: Server.provide(Account.info, async ({ capability, invocation }) => {
         const results = await ctx.kvs.accounts.get(capability.with)
         if (!results) {
-          throw new Failure('Account not found...')
+          return new Failure('Account not found.')
```
Comment (alanshaw) [inline packages/access-api/src/service/index.js:28]: It feels counter-intuitive to return an error...why can we not throw these?

File: packages/access-api/src/service/index.js [hunk 2]
```diff
@@ -46,21 +87,29 @@ export function service(ctx) {
...
             })
             .delegate()
 
           const encoded = await delegationToString(inv)
+          const url = `${ctx.url.protocol}//${ctx.url.host}/validate-email?ucan=${encoded}&mode=recover`
+
           // For testing
           if (ctx.config.ENV === 'test') {
-            return encoded
+            return url
```
Comment (alanshaw) [inline packages/access-api/src/service/index.js:112]: Only return in test?

**Review by alanshaw — APPROVED** [review]

File: packages/access-api/src/service/index.js (hunk 1 above)
Comment (hugomrdias) [reply packages/access-api/src/service/index.js:28]: These are known errors that should be returned to the user but not sent to sentry.
Ucanto supports multiple invocations in the same request thats why the responses are always arrays, so some invocations can succeed and some can error thats why we dont throw here.

In ucanto a known error like "account not found" is a valid response, the server types also include those. Any other exception error like "fail to parse json" that we want to send to sentry we should throw from the code.

File: packages/access-api/src/service/index.js (hunk 2 above)
Comment (hugomrdias) [reply packages/access-api/src/service/index.js:112]: yes just for tests so we can process the delegation without dealing with emails.

**Review by alanshaw — APPROVED** [review]

**Pull request merged**